src/Reference/Task3.java

```diff
@@ -0,0 +1,72 @@
+package Reference;
+
+import java.util.Scanner;
+
+public class Task3 {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter a number to determine the range of array (e.g. 2*2, 3*3, ...): ");
+        int dim = reader.nextInt();
+
+        if (dim < 2) System.out.println("Range of two-dimensional array cannot be less than 2");
+        else {
+            //initialization
+            int[][] array = new int[dim][dim];
+            for (int i = 0; i < dim; i++) {
+                for (int j = 0; j < dim; j++) {
+                    System.out.println("Enter the [" + i + ", " + j + "] element of array");
+                    array[i][j] = reader.nextInt();
+                }
+            }
+
+            //print 2d array
+            for (int i = 0; i < dim; i++) {
+                for (int j = 0; j < dim; j++) {
+                    System.out.print(array[i][j] + "\t");
+                }
+                System.out.println();
+            }
+
+            //sum of every row and col
+            int sum = 0, dSum = 0, lsum = 0, rsum = 0;
+            int[] colSum = new int[dim];
+            int[] rowSum = new int[dim];
+            int[] diagSum = new int[dim];
+
```

Comment:
Let's create a separate method for each calculation. 
A method that calculates sum of rows. A method for sum of columns and a method for diagonal lines